src/main/java/com/conveyal/gtfs/util/GeoJsonUtil.java

```diff
@@ -192,17 +198,20 @@ private static List<LocationShape> unpackLocationShapes(
         List<String> errors
     ) {
         ArrayList<LocationShape> locationShapes = new ArrayList<>();
+        Set<String> seenLocationIds = new HashSet<String>();
```

Comment:
Replace constructor with `<>` operator. I.e. `new HashSet<>();`

Reply:
Updated in https://github.com/conveyal/gtfs-lib/pull/374/commits/a573a3774be4d8dbc744e22d3e6ccec3d5e99593